src/main/java/hanziToAnki/chinese/ChineseDeckStyler.java

```diff
@@ -2,12 +2,19 @@
 
 import hanziToAnki.DeckStyler;
 import hanziToAnki.Word;
-import java.util.List;
-import java.util.Set;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class ChineseDeckStyler implements DeckStyler {
     private static final String DELIMITER = "\t";
     private static final String CLOSING_HTML_TAG = "</span>";
+    private static final List<String> HEADERS = Arrays.asList(
+            "#notetype:Chinese",
```

Comment:
this means that if the user already has a note type named "Chinese" (as I seem to), an import will automatically use that note type